CONTRIBUTING.md

```diff
@@ -0,0 +1,113 @@
+- Is the change as small as possible
+- Should it be chopped up into smaller chunks
+- Is the cost of reviewing the change worth the improvement made by the change
+Again, some PR’s are too big or provide too little value to merge.
+
+
+### Resources to help you get started
+
+Here are some resources to help you get started on how to contribute code or new content.
+
+- [Developers Guide](https://github.com/dotnet/fsharp/blob/main/DEVGUIDE.md) to get started on building the source code on your own.
+- [Test Guide](https://github.com/dotnet/fsharp/blob/main/TESTGUIDE.md) how to build run and work with test cases.
+- [F# compiler guide](https://github.com/dotnet/fsharp/blob/main/docs/index.md)
+- [F# language specification](https://fsharp.org/specs/language-spec/)
+- [F# language design](https://github.com/fsharp/fslang-design/)
+- [F# language suggestions](https://github.com/fsharp/fslang-suggestions/)
+- [help wanted](https://github.com/dotnet/fsharp/labels/help%20wanted) where to start
+
+### Submitting a pull request
+
+You will need to sign a [Contributor License Agreement](https://cla.dotnetfoundation.org/) when submitting your pull request. To complete the Contributor License Agreement (CLA), you will need to follow the instructions provided by the CLA bot when you send the pull request. This needs to only be done once for any .NET Foundation OSS project.
+
+If you don't know what a pull request is read this article: <https://help.github.com/articles/using-pull-requests>. Make sure the repository can build and all tests pass. Familiarize yourself with the project workflow and our coding conventions. 
+
+- **DO** ensure submissions pass all Azure DevOps legs and are merge conflict free.
+- **DO** submit language feature requests as issues in the [F# language](https://github.com/fsharp/fslang-suggestions) repos.  Please note: approved in principle does not guarantee acceptance.
+- **DO NOT** submit language features as PRs to this repo first, or they will likely be declined.
+- **DO** submit issues for other features. This facilitates discussion of a feature separately from its implementation, and increases the acceptance rates for pull requests.
+- **DO NOT** submit large code formatting changes without discussing with the team first.
+
+### Reviewing pull requests
+
+Our repository gets a high volume of pull requests and reviewing each of them is a significant time commitment. Our team priorities often force us to focus on reviewing a subset of the active pull requests at a given time.
+
+### Feedback
+
+Your pull request will now go through extensive checks by the subject matter experts on our team. Please be patient; we have hundreds of pull requests across all of our repositories. Update your pull request according to feedback until it is approved by one of the Roslyn team members. After that, one of our team members may adjust the branch you merge into based on the expected release schedule.
```

Comment:
Can you please clarify:

> Update your pull request according to feedback until it is approved by one of the Roslyn team members.

 So now we will need an extra review from  `Roslyn team` ? apart from the F# Team ?

Reply:
No ... copy paste error.